### PR TITLE
Update student list

### PR DESCRIPTION
**Pull request by hgrehm**

### CODE REVIEW

File: people/people.yml [hunk 1]
```diff
@@ -1258,6 +1272,7 @@
   level: Project Member
```
Comment (ntlhui) [inline people/people.yml]: Should be Project Member - Alumni

File: people/people.yml [hunk 2]
```diff
@@ -1305,38 +1320,35 @@
...
```
Comment (ntlhui) [inline people/people.yml]: Should be Project Member - Alumni

File: people/people.yml [hunk 3]
```diff
@@ -1305,38 +1320,35 @@
...
   name: Pranjal Chatterjee
   start: 2022
+  end: 2023
 - description: Smartfin since 2022, 2022 Summer REU
   expedition: false
   image: https://e4e.ucsd.edu/wp-content/uploads/reu2022_agarde-200x200.jpg
   level: Project Lead
   name: Anshul Garde
   start: 2021
```
Comment (ntlhui) [inline people/people.yml]: add end?

File: people/people.yml [hunk 4]
```diff
@@ -1394,18 +1406,22 @@
   description: Acoustic Species ID since 2022
   level: Project Candidate
   start: 2022
+  end: 2023
 - name: Joe Yan
   description: Acoustic Species ID since 2022
   level: Project Member
```
Comment (ntlhui) [inline people/people.yml]: Alumni

File: people/people.yml [hunk 5]
```diff
@@ -1442,10 +1461,11 @@
   description: Aye-Aye Sleep Monitoring since 2022
   level: Project Member
```
Comment (ntlhui) [inline people/people.yml]: Alumni

File: people/people.yml [hunk 6]
```diff
@@ -1471,6 +1491,7 @@
   description: Mangrove Monitoring since 2022
   level: Project Member
```
Comment (ntlhui) [inline people/people.yml]: Alumni?

File: people/people.yml [hunk 7]
```diff
@@ -1506,7 +1528,7 @@
   level: Project Candidate
   start: 2022
 - name: Kewei Chen
-  description: Smartfin since 2023
+  description: Smartfin since 2022
   level: Project Candidate
   start: 2022
```
Comment (ntlhui) [inline people/people.yml]: Alumni?

File: people/people.yml [hunk 8]
```diff
@@ -1684,54 +1709,78 @@
...
   start: 2023
 - name: Rana Singh
   description: Hardware Team supporting Smartfin since 2023
-  level: Project Candidate
+  level: Project Member
   start: 2023
+  image: https://e4e.ucsd.edu/wp-content/uploads/2023_rsingh.jpg
 - name: Ela Lucas
   description: Mechanical Engineer supporting Radio Telemetry Tracking since 2023
   level: Project Member
```
Comment (ntlhui) [inline people/people.yml]: Alumni?

File: people/people.yml [hunk 9]
```diff
@@ -1684,54 +1709,78 @@
...
   start: 2023
+  end: 2023
 - name: Nicholas Reyes
   description: Radio Telemetry Tracking
   level: Project Member
-  start: 2023 
+  start: 2023
+  end: 2023
```
Comment (ntlhui) [inline people/people.yml]: Alumni?

File: people/people.yml [hunk 10]
```diff
@@ -1684,54 +1709,78 @@
...
 - name: Derick Mathews
   description: Software for Wireless Underwater Communication System on Volt since 2023
   level: Project Member
```
Comment (ntlhui) [inline people/people.yml]: Alumni?

File: people/people.yml [hunk 11]
```diff
@@ -1684,54 +1709,78 @@
...
   start: 2023
+  end: 2023
 - name: Hamish Grant
   description: Fishsense, Summer 2023
   level: Project Member
```
Comment (ntlhui) [inline people/people.yml]: Alumni?

File: people/people.yml [hunk 12]
```diff
@@ -1684,54 +1709,78 @@
...
   start: 2023
+  end: 2023
 - name: Emily Thorpe
   description: Smartfin since 2023
   level: Project Member
```
Comment (ntlhui) [inline people/people.yml]: Alumni?

File: people/people.yml [hunk 13]
```diff
@@ -1684,54 +1709,78 @@
...
   start: 2023
+  end: 2023
 - name: Josephine Dominguez
   description: Smartfin, Radio Telemetry Tracking, and FishSense since 2023
   level: Project Member
```
Comment (ntlhui) [inline people/people.yml]: Alumni?

File: people/people.yml [hunk 14]
```diff
@@ -1684,54 +1709,78 @@
...
   start: 2023
+  end: 2023
 - name: Michael Pham
   description: Mangrove Monitoring since 2023
   level: Project Member
```
Comment (ntlhui) [inline people/people.yml]: Alumni?

File: people/people.yml [hunk 15]
```diff
@@ -1684,54 +1709,78 @@
...
   start: 2023
+  end: 2023
 - name: Jordan Reichhardt
   description: Smartfin, Radio Telemetry Tracking, Fishsense since 2023
   level: Project Member
```
Comment (ntlhui) [inline people/people.yml]: Alumni?

File: people/people.yml [hunk 16]
```diff
@@ -1684,54 +1709,78 @@
...
   start: 2023
+  end: 2023
 - name: Ana Perez
   description: FishSense since 2023
   start: 2023
+  end: 2023
   level: Project Member
```
Comment (ntlhui) [inline people/people.yml]: Alumni?

File: people/people.yml [hunk 17]
```diff
@@ -1684,54 +1709,78 @@
...
 - name: Chima Nwughala
   description: AKER, Summer 2023
   level: Project Member
```
Comment (ntlhui) [inline people/people.yml]: Alumni?

File: people/people.yml [hunk 18]
```diff
@@ -1684,54 +1709,78 @@
...
   start: 2023
+  end: 2023
 - name: Diana Ekechukwu
   description: Junkyard Computing, Summer 2023
   level: Project Member
```
Comment (ntlhui) [inline people/people.yml]: Alumni?

**Review by ntlhui — CHANGES_REQUESTED** [review]

File: people/people.yml (hunk 4 above)
Comment (hgrehm) [reply people/people.yml]: Student list hasn't previously had "Alumni" label added to Project Candidates who have left E4E. Do we want to update all previous candidates to specify Alumni? (Do we even need to list candidates on our website?)

Comment (ntlhui) [reply people/people.yml]: Project Candidates don't show up on the website.  I guess if they've left without becoming a project member, we just drop them from the list - don't qualify them as alumni.

**Comment by ntlhui** [comment]
```
Traceback (most recent call last):
  File "/home/runner/work/website/website/people/create_people_page.py", line 334, in <module>
Dr. Albert Lin's end year might be wrong!
    main()
  File "/home/runner/work/website/website/people/create_people_page.py", line 222, in main
    file.write(alumni_html)
UnicodeEncodeError: 'ascii' codec can't encode character '\u201c' in position 45093: ordinal not in range([12](https://github.com/UCSD-E4E/website/actions/runs/6917947049/job/18819676656#step:5:13)8)
```

**Comment by ntlhui** [comment]
ok, issue is here: https://github.com/UCSD-E4E/website/blob/3e5bfda802d667c307736f7136beec1adba7415f/people/people.yml#L1077

We require ascii only, so we'll need to change this to \" instead.

The reason this hasn't popped earlier is because we drop the entry way before due to level.  We need to add a check early on to make sure everything encodes correctly, and to provide contextual clues as to where the encoding failed.

Fixed the entry in 14fcd92e52f9f3d034b0ee31321321007c6cf654

Issue to address checks in #53

**Comment by ntlhui** [comment]
Valli Nachiappan should be alumni
Aryaman Jadhav should be alumni
@TylerFlar isn't Ashwin Muralidharan no longer active?
Alexander Tang should be alumni

**Review by ntlhui — APPROVED** [review]

**Pull request merged**